recipes/mplleaflet/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.0.5" %}
+
+package:
+    name: mplleaflet
+    version: {{ version }}
+
+source:
+    fn: mplleaflet-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/m/mplleaflet/mplleaflet-{{ version }}.tar.gz
+    md5: cf8ec819862e4486f477fb71101477f1
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - jinja2
+        - mplexporter
+
+test:
+    imports:
+        - mplleaflet
+
+about:
+    home: https://github.com/jwass/mplleaflet
+    license: BSD-3-Clause
+    summary: Convert Matplotlib plots into Leaflet web maps
+
+extra:
+    recipe-maintainers:
+        - jwass
```

Comment:
@jwass I added you as a maintainer here. Let me know if that is OK or if I should remove your GitHub handle from the recipe. (In conda-forge that means you'll have merge rights to the recipe repository.)